packages/menu/src/Menu.spec.tsx

```diff
@@ -348,7 +364,7 @@ describe('packages/menu', () => {
 
       expect(firstItem).toHaveFocus();
 
-      userEvent.keyboard('[Enter]');
+      userEvent.type(menuEl!, '[Enter]');
```

Comment:
I'm just curious: why are we switching this to `.type`, but on L234, we're switching to `.keyboard`?